lang/java/reef-common/src/main/java/org/apache/reef/runtime/common/driver/context/EvaluatorContext.java

```diff
@@ -170,6 +170,26 @@ public synchronized void submitContext(final Configuration contextConfiguration)
     this.contextControlHandler.send(contextControlProto);
   }
 
+  public synchronized void submitContext(final String contextConf) {
```

Comment:
No, this is the submission flow from the .NET Driver to the .NET Evaluator, so the configuration has to be serialized. Java does not know how to interpret a .NET `Configuration`.